lib/scope.js

```diff
@@ -650,9 +650,34 @@ module.exports = {
     }
   },  // Ends scope.setLanguage()
 
-  normalizeTable: async function ( scope, { var_data, from_story_table=true }) {
-    /* Return data derived from cucumber variable-setting data.
-     * Table must have headers. Arrays are not supported. */
+  normalizeTable: async function ( scope, { var_data=null, raw_var_data=null, from_story_table=true }) {
+    /* Return data derived from cucumber variable-setting data, from either an array or object format */
```

Comment:
Maybe this should include a note about why some things would be passing in just `var_data` or something? Maybe not necessary, but it took me a little while to understand why that was sticking around.